patches/chrome-browser-safe_browsing-android-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/safe_browsing/android/BUILD.gn b/chrome/browser/safe_browsing/android/BUILD.gn
+index a2edc70556e9ac331758dfd4017d1e11a2b78b80..5211c15f00fe115015a8045616d8151f9edb6ad2 100644
+--- a/chrome/browser/safe_browsing/android/BUILD.gn
++++ b/chrome/browser/safe_browsing/android/BUILD.gn
+@@ -31,6 +31,7 @@ android_library("java") {
+     "java/src/org/chromium/chrome/browser/safe_browsing/settings/SafeBrowsingSettingsFragmentBase.java",
+     "java/src/org/chromium/chrome/browser/safe_browsing/settings/StandardProtectionSettingsFragment.java",
+   ]
++  sources += brave_browser_safe_browsing_android_sources
```

Comment:
cc @AlexeyBarabash can this be added using an existing patch?

Reply:
It seems not possible. But just in case I am trying to put `src/brave/browser/safe_browsing/android/java/src/org/chromium/chrome/browser/safe_browsing/settings/BUILD.gn` and specify its target as deps through `src/chrome/android/BUILD.gn` at `brave_chrome_java_deps`.

Reply:
@bridiver I was wrong, this is possible, maybe I received error in the past because I didn't set all the deps correctly. Draft is 
https://github.com/brave/brave-core/commit/f88540ddf0732c51e3275aa44299656535b8b4be

Doing the same for `brave_browser_search_engines_android_sources` for the comment below.

Reply:
@bridiver addressed in https://github.com/brave/brave-core/pull/8662